guides/working-with-us.md

```diff
@@ -0,0 +1,26 @@
+# Working With Us
```

Comment:
what if this was on the contact page?

i wrote this to external people who may want to work with us, but i don't expect those people to check our guides. or, maybe this could be re-framed as a guide to an idea contract, or something.